CleverPush/Source/CleverPushInstance.m

```diff
@@ -3037,6 +3037,10 @@ - (void)setAppBannerTrackingEnabled:(BOOL)enabled {
     [CPAppBannerModule setTrackingEnabled:enabled];
 }
 
+- (BOOL)getAppBannerDraftsEnabled {
```

Comment:
We also need one method like in Android: `setAppBannerDraftsEnabled`
https://github.com/cleverpush/cleverpush-android-sdk/commit/4603de8f9aa8a38d41dd416d1cda4731327c5437#diff-dadb76999f5ed7d5f6c363d011ef4b354183a8b5cee4ecd118f2152c882891f1R2781

Reply:
Hello @gbhrdt We have already added the methods for that.